pkg/queue/queue.go

```diff
@@ -72,14 +72,21 @@ type ClusterQueue struct {
 	QueueingStrategy kueue.QueueingStrategy
 
 	heap heapImpl
+
+	// unschedulableWorkloads holds workloads that have been tried and determined unschedulable.
+	unschedulableWorkloads *UnschedulableWorkloads
```

Comment:
```suggestion
	InadmissibleWorkloads *InadmissibleWorkloads
```

Reply:
We sort of adopted the terms admitted/inaddmissible rather than scheduled/unschedulable.

Reply:
Done